indico/modules/api/templates/user_profile.html

```diff
@@ -12,15 +12,17 @@
                 {% if key and can_modify %}
                     <div class="i-box-buttons toolbar">
                         <div class="group">
-                            <button type="button" class="i-button warning icon-plus" data-href="{{ url_for('.key_create') }}" data-method="post"
+                            <button type="button" class="ui orange button" data-href="{{ url_for('.key_create') }}" data-method="post"
```

Comment:
Maybe you could use this as an opportunity to wrap this line and put some of the properties in the next line?

Reply:
Wouldn't this imply changing it in all the files to keep it consistent? 
I think in that case we should enforce it by linting.

Reply:
I don't think there's a good linter tool that works with Jinja. Anyway, we usually wrap if the line becomes longer than 120 chars otherwise. No need to be super consistent with wrapping things everywhere though - just wrap it nicely in new code ;)

Reply:
Yeah, we don't have to update every file, just the files that introduce some changes.

Reply:
(but please don't start major reformattings of unchanged code parts, it makes reviewing PRs harder)